requirements/requirements.in

```diff
@@ -10,7 +10,7 @@ django-braces==1.13.0
 django-celery-results~=1.0
 django-crispy-forms==1.7.0
 jsonfield==1.0.3
-dropbox==7.3.1
+dropbox==9.3.0
```

Comment:
This is a pretty big jump. how sure are we that it works? (previous dropbox upgrades have been annoying, so i've been procratinating it)

Reply:
I haven't tested it.  It's in this PR because the current version blocks installing requirements.